### PR TITLE
gradel upgrade

### PR DESCRIPTION
**Pull request by kistitan**
Automatic gradle upgrade done by android studio

### CODE REVIEW

File: FetLife/.idea/gradle.xml [hunk 1]
```diff
@@ -5,15 +5,21 @@
       <GradleProjectSettings>
         <option name="distributionType" value="LOCAL" />
         <option name="externalProjectPath" value="$PROJECT_DIR$" />
-        <option name="gradleHome" value="$APPLICATION_HOME_DIR$/gradle/gradle-2.8" />
-        <option name="gradleJvm" value="1.7" />
+        <option name="gradleHome" value="C:\Program Files\Android\Android Studio\gradle\gradle-2.10" />
```
Comment (philippt) [inline FetLife/.idea/gradle.xml]: ...don't know if one should manually change that, but `$APPLICATION_HOME_DIR$` looks better to me than `C:\...`

**Comment by philippt** [comment]
:+1:

**Comment by reneklacan** [comment]
👍

**Comment by jacortinas** [comment]
:+1:

**Pull request merged**